app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, devise is supposed to ask for email confirmation. :confirmable should therefore be enabled.
![Screen Shot 2023-06-20 at 9 35 41 PM](https://github.com/rmrmrmr/RubyBlog/assets/62849872/9f290424-43d4-4af9-be1e-b24374724174)